### PR TITLE
Add full name near Sign out menu item

### LINKED ISSUE #70: Show name near `Sign Out (email)` link

### PR DESCRIPTION
**Pull request by eleidan**
Fixes #70

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -31,6 +31,13 @@ def days_since_employment
     Time.zone.today - employment_date + 1
   end
 
+  def full_name
+    result = []
+    result.push(first_name)  unless first_name.nil?
+    result.push(last_name)   unless last_name.nil?
+    result.join(' ')
```
Comment (NazarKY) [inline app/models/user.rb]: Maybe better to write something like `"#{first_name} #{last_name}".strip` ?

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -38,7 +38,7 @@
             %ul.nav.navbar-nav.navbar-right
               %li
                 - if user_signed_in?
-                  = link_to "Sign out (#{current_user.email})", destroy_user_session_path, method: :delete
+                  = link_to "Sign out (#{current_user.full_name}, #{current_user.email})", destroy_user_session_path, method: :delete
```
Comment (NazarKY) [inline app/views/layouts/application.html.haml]: Is it possible to make this line not so long, split it ?

File: app/models/user.rb (hunk 1 above)
Comment (eleidan) [inline app/models/user.rb]: No "maybe", it is just better, and it work!
Thank you!

File: app/views/layouts/application.html.haml (hunk 2 above)
Comment (eleidan) [inline app/views/layouts/application.html.haml]: I think it should stay as it is.

Comment (NazarKY) [inline app/views/layouts/application.html.haml]: ok let it be so - it's not important

**Pull request merged**